app/controllers/api/admin/green_lanes/themes_controller.rb

```diff
@@ -0,0 +1,33 @@
+module Api
+  module Admin
+    module GreenLanes
+      class ThemesController < AdminController
+        before_action :check_service, :authenticate_user!
+
+        def index
+          render json: serialize(themes.to_a)
+        end
+
+        private
+
+        def themes
+          @themes ||= ::GreenLanes::Theme.order(Sequel.desc(:id))
```

Comment:
This would make more sense as section and subsection

Reply:
done